modules/aws-secure-token-service.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-short-lived-tokens.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="amazon-secure-token-service_{context}"]
+= AWS Secure Token Service
```

Comment:
Title should be a gerund https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures